### PR TITLE
SimpleTree blog - 

### PR DESCRIPTION
**Pull request by hacke1978**
cannot finish a paragraph with a picture so asd-line is added three times.

I do not find any more errors. HTML looks fine except this. I was not sure about one more:

It looks like the other entries, which are infact all code sprints and it is also tagged like this.

About content. I tried to stick just to some "fency" PCL related stuff in my tool and avoided the more statistical/forestry related contents. In majority I just use screenshots taken from to open access publications.

### CODE REVIEW

File: blogweb/index.rst [hunk 1]
```diff
@@ -40,6 +40,9 @@ the logos below to go to the respective sprint blog page.
 .. image:: _static/gsoc14.png
   :target: http://www.pointclouds.org/blog/gsoc14
   :class: foo
+.. image:: _static/simple_tree.png
+  :target: http://www.pointclouds.org/blog/gsoc14
```
Comment (VictorLamoine) [inline blogweb/index.rst]: This is wrong, you want to point to `http://www.pointclouds.org/blogweb/simpletree`

File: blogweb/simpletree/hackenberg/index.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+:email: jan.hackenberg@posteo.de
+:project: SimpleTree
+:mentor: 
+
+About me
+--------
+
+I am currently a PhD candidate at the former Chair of Forest Growth (University of Freiburg). My research interests include
+geometrical tree modelling methods - in the field of computational forestry often reffered to as Quantitative Structure Modelling.
```
Comment (VictorLamoine) [inline blogweb/simpletree/hackenberg/index.rst]: There is no need for a carriage return here.

File: blogweb/simpletree/conf.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
+# -*- coding: utf-8 -*-
+#
+# HRCS Website documentation build configuration file, created by
```
Comment (VictorLamoine) [inline blogweb/simpletree/conf.py]: You might want to change HRCS to SimpleTree

File: blogweb/simpletree/conf.py [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+source_suffix = '.rst'
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'HRCS Website'
```
Comment (VictorLamoine) [inline blogweb/simpletree/conf.py]: Also here

File: blogweb/simpletree/conf.py [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+# If true, an OpenSearch description file will be output, and all pages will
+# contain a <link> tag referring to it.  The value of this option must be the
+# base URL from which the finished HTML is served.
+#html_use_opensearch = ''
+
+# This is the file name suffix for HTML files (e.g. ".xhtml").
+#html_file_suffix = None
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'HRCSWebsitedoc'
```
Comment (VictorLamoine) [inline blogweb/simpletree/conf.py]: And here

File: blogweb/simpletree/conf.py [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+# The paper size ('letter' or 'a4').
+#latex_paper_size = 'letter'
+
+# The font size ('10pt', '11pt' or '12pt').
+#latex_font_size = '10pt'
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title, author, documentclass [howto/manual]).
+latex_documents = [
+  ('index', 'HRCSWebsite.tex', u'HRCS Website Documentation',
```
Comment (VictorLamoine) [inline blogweb/simpletree/conf.py]: And here

File: blogweb/simpletree/conf.py [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+# If false, no module index is generated.
+#latex_domain_indices = True
+
+
+# -- Options for manual page output --------------------------------------------
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [
+    ('index', 'hrcswebsite', u'HRCS Website Documentation',
```
Comment (VictorLamoine) [inline blogweb/simpletree/conf.py]: And here :)

File: blogweb/simpletree/index.rst [hunk 8]
```diff
@@ -0,0 +1,45 @@
+SimpleTree Code Sprint
```
Comment (VictorLamoine) [inline blogweb/simpletree/index.rst]: It's not a code sprint so I would change the name here; maybe SimpleTree project fits better?

**Comment by VictorLamoine** [comment]
Hello Jan, thank you for taking the time to write a blog article as requested!

I've added some remarks in the files themselves, here are others:
-  `simple_tree.png` and `simpletree.png` looks like duplicated images, remove one of them.
- The files ending with `~` are probably Gedit backup files, we don't need/want them in the repository, delete them. You can delete them with `find . -iname "*~" -exec rm -i {} \;` in a terminal. You can disable this Gedit behavior in Edit > Preferences > Editor tab > Create a backup of files before saving.
- Is `blogweb/simpletree/hackenberg/files/final_report.pdf` available anywhere else on the web? Maybe it's a better idea to point to your website rather than uploading it here?
- You don't use `blogweb/simpletree/images/pcl-hrcs.png`, delete it.

How to edit your pull request?
---
Just in case you are a git/github beginner, here are some instructions on how to edit your pull request:
```
cd blog
# Make your modifications
git add .
git commit -m "I have updated something"
git push
```

The pull request will be automatically updated with the changes.
I think this is really good advertising for the PCL and for your work :+1:

**Comment by hacke1978** [comment]
Hi Victor

I will take care of everything tomorrow.

Jan

**Comment by hacke1978** [comment]
Thanks a lot Victor, you give great support.

I did the commit/push with the suggested improvements.

**Comment by VictorLamoine** [comment]
Looks good on my side.
@jspricke,what do you think?

**Pull request merged**